addons/mail/static/src/discuss/core/public_web/thread_actions.js

```diff
@@ -0,0 +1,32 @@
+import { threadActionsRegistry } from "@mail/core/common/thread_actions";
+import { SubChannelList } from "@mail/discuss/core/public_web/sub_channel_list";
+import { useChildSubEnv, useComponent } from "@odoo/owl";
+import { _t } from "@web/core/l10n/translation";
+import { usePopover } from "@web/core/popover/popover_hook";
+
+threadActionsRegistry.add("sub-channel", {
```

Comment:
`show-threads`, to match with `name` on UI

Reply:
Done too, but same remark as before: kind of sad not to keep the sub-channel wording as the thread one is quite confusing. Not a big deal anyway.